subaccounts.go

```diff
@@ -44,30 +49,30 @@ func (c *Client) SubaccountCreate(s *Subaccount) (res *Response, err error) {
 func (c *Client) SubaccountCreateContext(ctx context.Context, s *Subaccount) (res *Response, err error) {
 	// enforce required parameters
 	if s == nil {
```

Comment:
I may have a fundamental misunderstanding of something here. I see `err` getting set in a lot of places here but I am not quite clear how that is exposed to the caller of this fuction.